src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,109 @@
+package duke;
+
+/**
+ * Represents a user interface that deals with interaction with users.
+ */
+public class Ui {
+
+    /**
+     * Class constructor for Ui class.
+     */
+    public Ui() {
+
+    }
+
+    /**
+     * Handles what to show during Duke's startup.
+     *
+     * @return response of duke in string.
+     */
+    public static String showWelcome() {
+        return "Hello! I'm Duke\nWhat can I do for you?";
+    }
+
+    /**
+     * Handles user interface for loading error.
+     *
+     * @return response of duke in string.
+     */
+    public String showLoadingError() {
+        return "Storage file not found at specified path!";
+    }
+
+    /**
+     * Handles user interface for when adding new tasks.
+     *
+     * @param taskToAdd task to be shown on ui.
+     * @param tasks list of tasks to get number of existing tasks.
+     * @return response of duke in string.
+     */
+    public String showAddedTask(Task taskToAdd, TaskList tasks) {
+        return "Got it. I've added this task:\n"
+                + taskToAdd.toString() + "\n"
+                + "Now you have " + tasks.getLength() + " tasks in the list";
+    }
+
+    /**
+     * Handles user interface for deleting tasks.
+     *
+     * @param taskToRemove task to be shown on ui.
+     * @param tasks list of tasks to get number of existing tasks.
+     * @return response of duke in string.
+     */
+    public String showDeletedTask(Task taskToRemove, TaskList tasks) {
```

Comment:
suggesting a change of method name to deletedTaskString- as the UI will be outputting the deleted task string to the console, this naming will help prevent confusion in the future.

Reply:
Hmm. Makes sense thanks.